data_steward/cleaners/clean_cdr.py

```diff
@@ -88,7 +20,7 @@ def clean_rdr_dataset(project=None, dataset=None):
         LOGGER.info('Dataset is unspecified.  Using default value of:\t%s', dataset)
 
     LOGGER.info("Cleaning rdr_dataset")
-    _clean_dataset(project, dataset, rdr_consts.SQL_QUERIES)
+    rule_4.run_clean_rule_4(project, dataset)
```

Comment:
This change only allows you to run a single query instead of a set of queries.  There will need to be a function to gather the SQL statements.  Then the statements are sent to the `clean_dataset(project_id, dataset_id, statements_list)` function.  As this is set up, this file will have to be modified every time we want to add a rule.